png_cicp_chunk_proposal.md

```diff
@@ -0,0 +1,50 @@
+# Adding support for HDR imagery to the PNG format
+Authors (alphabetical): Chris Blume, Chris Seeger, Pierre-Anthony Lemieux
+
+Status: Draft
+
+## Problem to be solved
+The gAMA chunk of the Portable Network Graphics (PNG) format (specified in [PNG]) parameterized the transfer function of the image as a power law. As such, it cannot model the Reference PQ or HLG OTFs specified in [BT2100-1], which are commonly used for HDR images.
+
+This specification uses the existing iCCP chunk to unambiguously signal the color system of an image that uses the Reference PQ EOTF specified in [BT2100-1]. It also allows graceful processing by decoders that do not conform to this specification by recommending fallback values for the gAMA chunk, cHRM chunk, and embedded ICC profile.
+
+An existing W3C group note (https://www.w3.org/TR/png-hdr-pq/) specifies an approach which is limited: it supports signaling of only the ITU BT.2100 PQ EOTF and uses magic values in the iCCP chunk to signal color spaces.
+
+[H.273](https://www.itu.int/rec/T-REC-H.273/en) specifies a controlled vocabulary for the parameterization of
+color space information:
+* Color Primaries
+* Transfer Function
+* Matrix Coefficients (only required if essence is stored as Y’CbCr)
+
+Use [ITU-T Series H Supplement 19](https://www.itu.int/rec/T-REC-H.Sup19-201910-I) “recommendations” to signal the controlled vocabulary for image formats commonly used for baseband linear broadcasts or file-based Video-on-Demand(VOD) services.
+* Tables within this document summarize values most commonly used in broadcast for 47 different standards documents including H.273
+
+
+## Basic requirements
+* does not break current implementations
+* extensible signaling of color space based on H.273
+* does not require the presence of iCCP chunk and embedded ICC profiles
```

Comment:
Why is this a requirement?

Reply:
Filesize is one reason, a chunk with a 7 byte payload is smaller than an ICC profile with zlib compression.

Reply:
True, but it doesn't give rich fallback then.  Also doesn't align with the work going on at the ICC to consider a `CICP` tag.

Reply:
I'm not sure I follow.
You are suggesting it should require the other chunk? Would you mind helping me understand why it should be required to have both?

Would it be an acceptable fallback to allow both chunks but not require both chunks? (If so, we probably want to explicitly specify how to handle a file with both chuns.)

Reply:
Per the 2021-01-12 meeting:

>     - two challenges re: signaling colorspace in the ICC profile in contrast to signaling colorspace directly in the image:
>         - adds a level of indirection
>         - ICC profile can/is be easily stripped

I think HDR-in-PNG signaling should not be contingent on ICC profiles, although ICC profiles can be included, e.g. for fallback.

Reply:
But now that means that PNG has its own model for colorspace specification that differs from other formats (eg. JPEG)

Reply:
Also, what if the embedded ICC profile *has* the new tag for HDR - why not support that?

Reply:
> 
> 
> But now that means that PNG has its own model for colorspace specification that differs from other formats (eg. JPEG)

PNG also has its own PNG-specific way to signal sRGB, that doesn't use an ICC profile.
EXIF has an EXIF-specific way to signal either sRGB or Adobe 1998 RGB, without an ICC profile